angular/src/app/modules/retros/retro/retro-detail/retro-detail.component.ts

```diff
@@ -444,6 +451,57 @@ export class RetroDetailComponent
       }
     });
   }
+
+  showDialogMulti(
+    retroDetail: RetroDetailCreateEditDto,
+  ): void {
+    const {
+      id,
+      userId,
+      projectId,
+      positionId,
+      point,
+      note,
+      retroId,
+      userName,
+      branchId,
+      userLevel,
+      userType,
+      projectName,
+      pmId
+    } = retroDetail;
+    let item = {
+      id,
+      userId,
+      projectId,
+      positionId,
+      point,
+      note,
+      retroId,
+      userName,
+      branchId,
+      userLevel,
+      userType,
+      projectName,
+      pmId
+    } as RetroDetailCreateEditDto;
```

Comment:
Change to `let item = Object.assign({},retroDetail);` or use `lodash` to clone data.